bor/genesis_test.go

```diff
@@ -0,0 +1,67 @@
+package bor_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/maticnetwork/heimdall/app"
+	"github.com/maticnetwork/heimdall/bor"
+	"github.com/maticnetwork/heimdall/bor/types"
+	chSim "github.com/maticnetwork/heimdall/checkpoint/simulation"
+	hmTypes "github.com/maticnetwork/heimdall/types"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type GenesisTestSuite struct {
+	suite.Suite
+
+	app *app.HeimdallApp
+	ctx sdk.Context
+}
+
+// SetupTest setup necessary things for genesis test
+func (suite *GenesisTestSuite) SetupTest() {
+	suite.app, suite.ctx, _ = createTestApp(true)
+}
+
+func TestGenesisTestSuite(t *testing.T) {
+	suite.Run(t, new(GenesisTestSuite))
+}
+
+func (suite *GenesisTestSuite) TestInitExportGenesis() {
+	t, app, ctx := suite.T(), suite.app, suite.ctx
+	spancount := 5
+	params := types.DefaultParams()
+	spans := []*hmTypes.Span{}
+	chainID := "15001"
+	start := uint64(0)
+	end := uint64(0)
+	chSim.LoadValidatorSet(4, t, app.StakingKeeper, ctx, false, 10)
+	app.BorKeeper.SetParams(ctx, types.DefaultParams())
+	valSet := app.StakingKeeper.GetValidatorSet(ctx)
```

Comment:
`LoadValidatorSet` above returns the validators set, any reason we cant use that directly?